lib/install.ps1

```diff
@@ -355,7 +355,17 @@ function args($config, $dir) {
 function run($exe, $arg, $msg, $continue_exit_codes) {
     if($msg) { write-host $msg -nonewline }
     try {
-        $proc = start-process $exe -wait -ea stop -passthru -arg $arg
+        #Check if arguments were provided in the manifest, and pass them to the installer if present
+        if ($arg)
```

Comment:
After resetting scoop to the current version (before this pull request, that is), this is the error that I get:

<img width="972" alt="screen shot 2016-03-27 at 04 01 19" src="https://cloud.githubusercontent.com/assets/11891085/14064423/bff597b6-f3d0-11e5-839a-1778cfe1370f.png">

I'm definitely a relative PowerShell newbie, but the error message above seems to indicate there's no way to include any null values in the arguments without this failing. I added the check for arguments as it was the simplest way to accomplish the correct result, but if there's a more elegant approach, I'm all for it.